dynamic-panel-transparency@rockon999.github.io/theming.js

```diff
@@ -150,6 +150,31 @@ function has_icon_shadow() {
 function remove_text_shadow() {
     Panel.remove_style_class_name('dpt-panel-text-shadow');
 }
+/**
```

Comment:
Remove `register_hide_corners` - you don't need a dynamic stylesheet for a static style.